timescaledb/how-to-guides/multinode-timescaledb/multinode-ha.md

```diff
@@ -1,34 +1,100 @@
-multiple data nodes. This natively replicates data at the chunk level, and
-protects against data node failures. It also provides high availability using an
-alternative to fully replicating each data node using.
-
-The data nodes themselves require no additional setup to use native replication,
-but the access node relies on streaming replication by default, so it requires
-some configuration to make it highly available.
-
-When data is written to a chunk, it uses the `replication_factor` setting to
-determine how many nodes it replicates the data to. By default, this is set to
-`1`, so there is no native replication. You can increase this number when you
-create the hypertable. For example, to replicate the data across a total of
-three data nodes:
+
+A multi-node installation of TimescaleDB can be made highly available
+in two different ways:
+
+1. by setting up one or more standbys for each node in the cluster, or
+2. by natively replicating data at the chunk level.
+
+The first option relies on streaming replication and the setup is
+similar to [configuring single-node HA][single-ha], although the
+configuration needs to be applied to each node independently.
+
+The second option relies on built-in capabilities of multi-node
+TimescaleDB to replicate data at the chunk level instead of
+replicating entire data nodes. The access node still relies on a
+streaming replication standby, but the data nodes need no additional
+configuration. Instead, the existing pool of data nodes share
+responsibility to host chunk replicas in order to be able to sustain
+node failures.
+
+There are pros and cons of each approach. Setting up standbys for each
+node in the cluster ensures that standbys are identical at the
+instance level and is a tried and tested method to provide high
+availability. However, it is also a heavy-weight solution and
+essentially requires setting up a mirror cluster.
+
+Native replication, on the other hand, offers simpler configuration
+and additional, built-in, capabilities, like being able to easier add
+and remove data nodes, and use different replication factors on each
+distributed hypertable. The downside, however, is that only chunks are
+replicated on the data nodes.
+
+The rest of this section focuses on native replication. To setup
+standbys for each node, follow the instructions for [single node
+HA][single-ha].
+
+## Native replication
+
+Native replication is a set of capabilities and APIs around which one
+can build a highly available multi-node TimescaleDB installation. At
+the core of native replication is the ability to write copies
+(replicas) of a chunk to multiple data nodes. Thus, a failure of one
+data node should not lead to loss of data as copies exist on other
+data nodes. If a data node is permanently lost, a new data node can be
+added to the cluster, and lost chunk replicas can be copied
+(re-replicated) from other data nodes to again reach the number of
+desired chunk replicas.
```

Comment:
```suggestion
## Native replication
Native replication is a set of capabilities and APIs that allow you to 
build a highly available multi-node TimescaleDB installation. At
the core of native replication is the ability to write copies or replicas
of a chunk to multiple data nodes. If one data node fails, copies exist
on other data nodes, so that data is not lost. If a data node is 
permanently lost, a new data node can be added to the cluster, 
and lost chunk replicas can be copied or re-replicated from 
other data nodes to reach the number of desired chunk replicas.
```